airbyte-integrations/connectors/source-opsgenie/integration_tests/configured_catalog.json

```diff
@@ -0,0 +1,63 @@
+{
+  "streams": [
+    {
+      "stream": {
+        "name": "teams",
+        "json_schema": {},
+        "supported_sync_modes": ["full_refresh"]
+      },
+      "sync_mode": "full_refresh",
+      "destination_sync_mode": "overwrite"
+    },
+    {
+      "stream": {
+        "name": "integrations",
+        "json_schema": {},
+        "supported_sync_modes": ["full_refresh"]
+      },
+      "sync_mode": "full_refresh",
+      "destination_sync_mode": "overwrite"
+    },
+    {
+      "stream": {
```

Comment:
You seem to have removed `alerts` stream (see below original code from a copy of the branch i had locally from your last PR). But `alerts` stream is seen in the code base. Either add the json and tests for this back or remove altogether.

```py
    {
      "stream": {
        "name": "alerts",
        "json_schema": {},
        "supported_sync_modes": ["full_refresh", "incremental"],
        "source_defined_cursor": true,
        "default_cursor_field": ["updatedAt"],
        "source_defined_primary_key": [["id"]]
      },
      "sync_mode": "incremental",
      "destination_sync_mode": "append",
      "cursor_field": ["updatedAt"]
    },
```

Reply:
I took the alerts out of the testing because the account I'm using for testing has a LOT of alerts going back several years, causing the tests to run for a long time and/or time out.

I'll put it back in and take a look at the alerts stream code. I think it should take the configured start date into account.

Reply:
Reintroduced alerts testing. No need to remove.